knowledge-base/unable-to-get-report-parameters-report-cannot-be-resolved.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Unable to get report parameters - Report reportName cannot be resolved
+description: Unable to get report parameters - Report reportName cannot be resolved
+type: troubleshooting
+page_title: Unable to get report parameters - Report reportName cannot be resolved
+slug: unable-to-get-report-parameters-report-cannot-be-resolved
+position: 
+tags: Azure, Linux
+ticketid: 1532722
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+Trying to display a report in a report viewer through the use of a [Reporting web service](../telerik-reporting-rest-conception) may result in the following error being displayed inside the report viewer.
+
+## Error Message
+
+```
+Unable to get report parameters.
+Report reportName.trdp/x cannot be resolved.
+```
+
+## Cause\Possible Cause(s)
+
+- The report definition file cannot be resolved by the reporting service because it cannot be found at the given path. 
+
+- The service does not have permissions for the folder used to store the report definitions and thus the reports cannot be resolved.
+
+- If the application with the reporting web service is deployed on Azure with **lower** than **Basic** [App Service plan](https://azure.microsoft.com/en-us/pricing/details/app-service/windows/).
```

Comment:
These are not causes. The problem occurs more frequently in these environments due to their limitations. For example, due to restricted GDI.